cmd/kube-batch/app/options/options.go

```diff
@@ -49,6 +61,7 @@ func (s *ServerOption) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.SchedulerName, "scheduler-name", "kube-batch", "kube-batch will handle pods with the scheduler-name")
 	fs.StringVar(&s.SchedulerConf, "scheduler-conf", "", "The namespace and name of ConfigMap for scheduler configuration")
 	fs.StringVar(&s.SchedulePeriod, "schedule-period", "1s", "The period between each scheduling cycle")
+	fs.StringVar(&s.PdbQueue, "pdb-queue", "", "The name of the Queue object to be used with PDBs instead of their namespace name")
```

Comment:
Comparing to add a parameter, prefer to add an annotation to PDB for `queue`.

Reply:
for example `scheduling.k8s.io/queue-name`

Reply:
with annotation, you can create more queues :)

Reply:
The specific use case I'm looking at is co-scheduling tf-operator jobs with other multi-node workloads (PodGroup/Queue based). Fully agree that annotations are more flexible, however if we go that route then tf-operator needs to be extended to add said annotations to its PDBs (either directly or via a mutating admission controller) which is problematic for existing deployments.

So we could consider this PR as a stop gap for environments where PDBs cannot be easily equipped with additional annotations (for whatever reason). 

Separately the longer term strategy should be decided: either introduce the annotations and extend the tf-operator's PDBs to include them or switch the latter to PodGroups entirely ?

Reply:
oh, for `tf-operator` case, I'd like to switch it to `PodGroups`; but we need to align with tf-operator team and sig-scheduling for it. How about open an issue in `kubeflow/tf-operator` for it firstly?

Reply:
btw, you did not use multiple queues, right?

Reply:
if we switch to `PodGroup`, we still need to provide default queue name for your case (similar to this PR); otherwise, tf-operator need to have a field for `Queue`.

`Queue` is an experimental feature right now, which means you can have a try, but other community should not dependent on it.

Because of above reasons, maybe your PR is better as a temp solution; WDYT?

Reply:
BTW, prefer `DefaultQueue` instead of `PdbQueue`.

Reply:
makes sense, will amend